code/modules/client/preference/preferences.dm

```diff
@@ -1797,6 +1803,32 @@ GLOBAL_LIST_INIT(special_role_times, list( //minimum age (in days) for accounts
 
 						flavor_text = msg
 
+				if("ipcloadouts")
+					var/choice
+					var/datum/robolimb/R = new()
+					var/robolimb_companies = list()
+					var/rparts = list("chest", "groin", "head", "r_arm", "r_hand", "r_leg", "r_foot", "l_leg", "l_foot", "l_arm", "l_hand")
+					for(var/comp in typesof(/datum/robolimb))	//This loop populates a list of companies that shells
+						R = new comp()
+						if(R.has_subtypes != 0)					//No Monitor Models
+							robolimb_companies[R.company] = R
+					R = new() //Re-initialize R.
+					choice = input(user, "Which manufacturer model would you like to use?") as null|anything in robolimb_companies
+					if(!choice)
+						return
+					R.company = choice
+					R = GLOB.all_robolimbs[R.company]
+					for(var/limb in rparts)
+						if(limb == "head")
+							ha_style = "None"
+							alt_head = null
+							h_style = GLOB.hair_styles_public_list["Bald"]
+							f_style = GLOB.facial_hair_styles_list["Shaved"]
+							m_styles["head"] = "None"
+						rlimb_data[limb] = choice
+						organ_data[limb] = "cyborg"
+					r_company = choice
```

Comment:
`r_company` is set, but it doesn't have to be set at all as it doesn't really map to anything.